lib/srv/sess.go

```diff
@@ -1773,8 +1773,12 @@ func (s *session) trackSession(ctx context.Context, teleportUser string, policyS
 		svc = nil
 	}
 
+	// apply shorter timeout to make opening ssh sessions more responsive during periods when auth service is down
+	timeoutCtx, timeoutCancel := context.WithTimeout(ctx, time.Second*5)
```

Comment:
Couldn't find a suitable value in the `defaults` package or any configuration value accessible from here, hence the hardcoded timeout.